### PR TITLE
Updating xUnit to v2.1.0 RTM

### PR DESCRIPTION
**Pull request by tannergooding**

### CODE REVIEW

**Comment by tannergooding** [comment]
FYI. @jmarolf, @heejaechang, @ManishJayaswal, @Pilchie, @jaredpar 

This is a supplement to the changes Jared completed over the weekend and brings us onto the stable RTM release.

**Comment by jaredpar** [comment]
:+1:

File: build/Targets/Roslyn.Toolsets.Xunit.targets [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup>
     <StartAction Condition="'$(StartActions)' == ''">Program</StartAction>
-    <StartProgram Condition="'$(StartProgram)' == ''">$(NuGetPackagesPath)\xunit.runner.console.2.1.0-beta4-build3109\tools\xunit.console.x86.exe</StartProgram>
+    <StartProgram Condition="'$(StartProgram)' == ''">$(NuGetPackagesPath)\xunit.runner.console.2.1.0\tools\xunit.console.x86.exe</StartProgram>
```
Comment (Pilchie) [inline build/Targets/Roslyn.Toolsets.Xunit.targets]: Is this file actually imported anymore?

File: build/Targets/Roslyn.Toolsets.Xunit.targets [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
     <StartArguments Condition="'$(StartArguments)' == ''">$(AssemblyName).dll -html $(OurDir)\xUnitResults\$(AssemblyName).html -noshadow</StartArguments>
```
Comment (Pilchie) [inline build/Targets/Roslyn.Toolsets.Xunit.targets]: Can we get rid of "-noshadow" everywhere, and push it into the .config files?

**Comment by Pilchie** [comment]
Generally +1

File: build/Targets/Roslyn.Toolsets.Xunit.targets (hunk 1 above)
Comment (tannergooding) [inline build/Targets/Roslyn.Toolsets.Xunit.targets]: Yes.

File: build/Targets/Roslyn.Toolsets.Xunit.targets (hunk 2 above)
Comment (tannergooding) [inline build/Targets/Roslyn.Toolsets.Xunit.targets]: That will part of a separate PR.

**Pull request merged**